packages/browser/src/browser/index.ts

```diff
@@ -209,12 +209,22 @@ async function registerPlugins(
     tsubMiddleware
   ).catch(() => [])
 
-  const inAppPlugin = options.integrations?.['Customer.io In-App Plugin'] as InAppPluginSettings
+  const inAppPluginName = 'Customer.io In-App Plugin'
+  let inAppPluginSettings = mergedSettings[
+    inAppPluginName
+  ] as InAppPluginSettings
+  if (!inAppPluginSettings) {
```

Comment:
🤔 is this needed? `mergedSettings` will already have `options.integrations[inAppPluginName]` if not overridden, right?

Reply:
I tested with `mergedSettings` only, but if the server doesn't return anything, the contents of options will be completely ignored.